NetSDK/Version.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Splitio
+{
+    public static class Version
+    {
+        public static string SplitSdkVersion = "0.0.1";
+        public static string SplitSpecVersion = "1.0";
```

Comment:
Lets talk about this class because I feel we have duplicated information between the nuget release and this file.